cli/commands.go

```diff
@@ -71,7 +71,7 @@ func Commands(metaPtr *meta.Meta) map[string]cli.CommandFactory {
 				CredentialBackends: map[string]logical.Factory{
 					"approle":  credAppRole.Factory,
 					"cert":     credCert.Factory,
-					"aws-ec2":  credAwsEc2.Factory,
```

Comment:
I don't remember saying that (maybe it was Vishal?), but, it just means it's not something I have strong feelings about :-)